### PR TITLE
fe: reload frontend from fum-file

### PR DESCRIPTION
**Pull request by michaellilltokiwa**

### CODE REVIEW

File: tests/partial_application/partial_application.fz [hunk 1]
```diff
@@ -59,17 +59,19 @@ partial_application is
...
   test "data.map (!!)" (data.map (!!)) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
-  test "data.map (!)" (data.map (!)) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
-  test "data.map (!) " (data.map (!) ) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
+  # NYI should use integer.postfix ! but does not find feature currently
+  # test "data.map (!)" (data.map (!)) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
+  test "data.map (!!!)" (data.map (!!!)) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
+  test "data.map (!!!) " (data.map (!!!) ) "[1, 2, 6, 24, 120, 720, 5040, 40320, 362880, 3628800]"
   test "((u64 11)..20).map (!)" (((u64 11)..20).map (!)) "[39916800, 479001600, 6227020800, 87178291200, 1307674368000, 20922789888000, 355687428096000, 6402373705728000, 121645100408832000, 2432902008176640000]"
   test "((u64 11)..15).map ( !)" (((u64 11)..15).map ( !)) "[39916800, 479001600, 6227020800, 87178291200, 1307674368000]"
   test "((u64 11)..15).map (! )" (((u64 11)..15).map (! )) "[39916800, 479001600, 6227020800, 87178291200, 1307674368000]"
```
Comment (fridis) [inline tests/partial_application/partial_application.fz:77]: This seems unrelated.

Comment (michaellilltokiwa) [reply tests/partial_application/partial_application.fz:77]: @fridis 
With this patch, an error would be raised for duplicate feature because of:
integer.postfix !
i32.postfix !

File: src/dev/flang/ast/Types.java [hunk 2]
```diff
@@ -283,6 +283,82 @@ public Resolved(SrcModule mod, CreateType ct, AbstractFeature universe)
           res.resolveTypes(t.feature());
         }
     }
+    public static interface LookupFeature
+    {
+      AbstractFeature lookupFeature(AbstractFeature outer, FeatureName fn);
+    }
+    public Resolved(LookupFeature luf, AbstractFeature _universe)
```
Comment (fridis) [inline src/dev/flang/ast/Types.java:290]: Can `Resolved(SrcModule mod, CreateType ct, AbstractFeature universe)` be implemented using this new method to avoid code duplication?

**Review by fridis — COMMENTED** [review]
just first comments, need to review again tomorrow.

File: src/dev/flang/ast/Types.java (hunk 2 above)
Comment (michaellilltokiwa) [reply src/dev/flang/ast/Types.java:214]: @fridis I removed the code duplication in a recent commit.
